### PR TITLE
Replace f-string with "...".format(...)

### PR DESCRIPTION
**Pull request by haypo**
Change adding Python 3.5 support.

### CODE REVIEW

**Comment by haypo** [comment]
Alternate syntax of PR #143, again, to add Python 3.5 compatibility.

File: blurb/blurb.py [hunk 1]
```diff
@@ -495,10 +495,10 @@ def _parse_next_filename(filename):
         components = filename.split("/")
         section, filename = components[-2:]
         section = unsanitize_section(section)
-        assert section in sections, f"Unknown section {section}"
+        assert section in sections, "Unknown section {}".format(section)
 
         fields = [x.strip() for x in filename.split(".")]
-        assert len(fields) >= 4, f"Can't parse 'next' filename! filename {filename!r} fields {fields}"
+        assert len(fields) >= 4, "Can't parse 'next' filename! filename {!r} fields {}".format(filename, fields)
```
Comment (1st1) [inline blurb/blurb.py]: This line is surely longer than 79 characters.

Comment (haypo) [inline blurb/blurb.py]: The PEP 8 allows up to 120 characters. The file already contains lines longer than 120 characters (before my change) ;-)

**Comment by serhiy-storchaka** [comment]
Thank you for doing this @haypo!

**Comment by serhiy-storchaka** [comment]
Brett, could you please update the version on PyPI? Current version is installable on 3.5, but doesn't work.

**Comment by brettcannon** [comment]
@serhiy-storchaka sure, but I'm traveling ATM so I won't be able to get to it until I return home (no later than sometime next week).

**Comment by Mariatta** [comment]
Prior to releasing to PyPI, I think [this line](https://github.com/python/core-workflow/blob/master/blurb/flit.ini#L12) in flit.ini should be updated so it can work with Python < 3.6.

The readme still states "blurb's only dependency is Python 3.6+.", guess that should be updated too.

**Comment by brettcannon** [comment]
I updated the version number, tweaked the trove classifiers, fixed the README, pushed [1.0.1](https://pypi.org/project/blurb/1.0.1/) to PyPI, and then bumped the version number for future development.

**Comment by serhiy-storchaka** [comment]
Unfortunately
```
python3 -m pip install --user blurb
```
still installs version 1.0. And explicit specifying the version 1.0.1 doesn't work.
```
$ python3 -m pip install --user blurb==1.0.1
Collecting blurb==1.0.1
  Could not find a version that satisfies the requirement blurb==1.0.1 (from versions: 1.0)
No matching distribution found for blurb==1.0.1
```

**Pull request merged**